server/lib/systemd/pbench-server.service.default

```diff
@@ -9,9 +9,13 @@ Documentation = https://github.com/distributed-system-analysis/pbench
 After=network.target httpd.service
 
 [Service]
+user = pbench
+group = pbench
 Type = simple
+PIDFile = /opt/pbench-server/gunicorn.pid
```

Comment:
Can we move this `.pid` file to `/run/pbench-server.gunicorn.pid`?  Using `/run` is a bit more "standard", and prefixing it with `pbench-server` would avoid potential conflicts from other sub-systems.

Reply:
sounds good

Reply:
Interesting, I cant run pbench-server service as a pbench user and try to read a file from `/run` directory (or we need to give `pbench` user an access to `run` directory) or alternatively run pbench-server service as a root user (without specifying user and group in service file just like what we have now)

Reply:
Is this still a problem? Do we have to create the /run/pbench-server directory beforehand with the appropriate owner/group?